evap/staff/views.py

```diff
@@ -950,6 +951,45 @@ def course_create(request, semester_id):
     return render(request, "staff_course_form.html", dict(semester=semester, course_form=course_form, editable=True))
 
 
+@manager_required
+def course_copy(request, semester_id, course_id):
+    semester = get_object_or_404(Semester, id=semester_id)
+    course = get_object_or_404(Course, id=course_id, semester=semester)
+    course_form = CourseCopyForm(request.POST or None, instance=course)
+
+    if course_form.is_valid():
+        copied_course = course_form.save()
+        messages.success(request, _("Successfully copied course."))
+
+        inactive_users = UserProfile.objects.filter(
+            Q(contributions__evaluation__course__in=[copied_course], is_active=False)
+            | Q(courses_responsible_for__in=[copied_course], is_active=False)
```

Comment:
think we don't need to use `__in` if we use it with a list with a single element.